test/integration/visitor_can_edit_cart_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.